server/webapp/WEB-INF/rails.new/webpack/views/agents/agent_row_widget.js.msx

```diff
@@ -86,6 +91,9 @@ const AgentRowWidget = {
         <td key="checkbox">
           {selectAgentCheckbox}
         </td>
+        <td key="agentIcon">
+          <img class='agent-icon' src={ isElasticAgent ? agent.elasticAgentIcon() : ''} />
```

Comment:
Prefer to not render the image instead of an image with no src on it. See how `selectAgentCheckbox` is rendered for admins only.

This would require removing the CSS rule for empty `src` attribute.